src/components/Tabs/Tabs.tsx

```diff
@@ -57,6 +61,7 @@ export const Tabs: React.FC<TabsProps> = ({
             active={item.id === activeTab}
             className={clsx('ui-mx-2', tabClassName)}
             disabled={item.disabled}
+            id={item.id}
```

Comment:
could you change the id to something more unique like "tab-${item.id}"

Reply:
sure